donate/thunderbird/templates/payment/newsletter_signup.html

```diff
@@ -0,0 +1,10 @@
+{% extends "payment/newsletter_signup_master.html" %}
+
+{% load i18n %}
+
+{% block thank-you %}
+    <h1 class="heading heading--primary heading--bottom-margin">{% trans "Thank you for your generous gift" %}</h1>
+    <p>{% blocktrans %}
```

Comment:
And here too, also because it will then match an existing string and use that translation

```suggestion
    <p>{% blocktrans trimmed %}
```